tools/build_utils.py

```diff
@@ -218,7 +218,7 @@ def build_tensorflow(venv_dir, src_dir, artifacts_dir, target_arch, verbosity):
         "bazel",
         "build",
         "--config=opt",
-        "--config=v2",
+        "--config=v1",
```

Comment:
What is this config v1 vs v2 here?

Reply:
It's this guy:
https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/compat/v1/config
My understanding is for `TF v1.15.x` we should just stick with `/v1/config` and only use that for `TF v2.x` and better.

Reply:
Surprisingly for `TF v1.15.0` they only supported `--config=v2` 🤔 
But then they realized they need `--config=v1` support and they added it to newer releases of `TF v1.15.x` so hence this change.